src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div
```

Comment:
Should we use `<li>` instead (also change its wrapper to `<ul>` in `KMultiselect.vue`)? I think `KSelect` is using `<ul>`/`<li>` elements.

Reply:
This PR also switches `KSelect` away from the `ul/li` structure. We keep facing issues when slotting and adding additional elements (e.g. a divider) that using `ul/li` makes very limited. By switching everything to `div` we get around the HTML syntax issues we've been seeing.

We could add `role="list"` and `role="listitem"` on the elements though; @kaiarrowood this might be a good addition.